www/compose.py

```diff
@@ -22,6 +22,9 @@
     ("Download", OrderedDict([
         ("download", "Download"),
     ])),
+    ("Users contributions", OrderedDict([
+		("contrib", "Users contributions"),
+    ])),
     # ("Wiki", "http://wiki.herbstluftwm.org"),
```

Comment:
Can you please indent with spaces instead of tabs? then, flake8 in the CI will also be happy:
```
./www/compose.py:26:1: W191 indentation contains tabs
./www/compose.py:26:1: E101 indentation contains mixed spaces and tabs
./www/compose.py:26:3: E126 continuation line over-indented for hanging indent
ERROR: InvocationError for command /__w/herbstluftwm/herbstluftwm/.tox/flake8/bin/flake8 . (exited with code 1)
```